scipy/optimize/optimize.py

```diff
@@ -433,7 +434,17 @@ def _minimize_neldermead(func, x0, args=(), callback=None,
     fatol : number, optional
         Absolute error in func(xopt) between iterations that is acceptable for
         convergence.
+    adaptive : bool, optional
+        Adapt algorithm parameters to dimensionality of problem. Useful for
+        high-dimensional minimization [1].
```

Comment:
I think this needs to be `[1]_` to link properly

Reply:
Right, thanks!